tests/Doctrine/Tests/Common/Annotations/AbstractReaderTest.php

```diff
@@ -12,31 +12,31 @@
 {
     public function getReflectionClass()
     {
-        return new ReflectionClass(DummyClass::class);
+        return new ReflectionClass(DummyClass::class); #Tests\Common\Annotations\DummyClass
     }
 
     public function testAnnotations()
     {
-        $class = $this->getReflectionClass();
+        $refClass = $this->getReflectionClass();
```

Comment:
Why the rename?

Reply:
Thank you for many suggesting. 
refClass (reflection class not normal class) for easy readable.

Reply:
Then do remove the abbreviation instead.

Reply:
Ok, I turn it back since there are many case like this. So I think we should assume $class ~ $reflectionClass